apps/nextjs-website/src/components/Hero.tsx

```diff
@@ -0,0 +1,14 @@
+'use client';
+import React from 'react';
+import { Hero as HeroEC } from '@pagopa/pagopa-editorial-components';
+import { HeroProps } from '@pagopa/pagopa-editorial-components/dist/components/Hero/index';
+
+const Hero: React.FC<HeroProps & { sectionID: string | undefined }> = (
```

Comment:
Since we're going to do this (`& { sectionID: string | undefined }`) for all sections, should we create a function which takes any type in input and returns the same type with the addition of `& { sectionID: string | undefined }`?

Reply:
If we can generalize a function that we can import where we need each time instead of writing the same code multiple times, I'm always for the second option. Better create the function than having cloned rows multiple times